DefInjected/ThingDef/Buildings_Power.xml

```diff
@@ -2,14 +2,14 @@
 <LanguageData>
   
   <!-- EN: battery -->
-  <Battery.label>baterie</Battery.label>
+  <Battery.label>akumulator</Battery.label>
   <!-- EN: Stores electricity for later use. Charged batteries explode when exposed to rain or fire. -->
-  <Battery.description>Gromadzą energię, gdy jest jej nadmiar i pozwalają wykorzystać, gdy jej brakuje. Uwaga - mają w zwyczaju wybuchać, jeśli są wystawione na ogień lub deszcz.</Battery.description>
+  <Battery.description>Gromadzi energię do późniejszego użytku. Uwaga - naładowane akumulatory mają w zwyczaju wybuchać, jeśli są wystawione na ogień lub deszcz.</Battery.description>
   
   <!-- EN: chemfuel powered generator -->
-  <ChemfuelPoweredGenerator.label>generator na paliwo</ChemfuelPoweredGenerator.label>
+  <ChemfuelPoweredGenerator.label>generator na chemiopaliwo</ChemfuelPoweredGenerator.label>
```

Comment:
```suggestion
  <ChemfuelPoweredGenerator.label>generator na paliwo chemiczne</ChemfuelPoweredGenerator.label>
```
może taka forma ?

Reply:
W takim razie czeka mnie przeoranie tego we wszystkich plikach z chemiopaliwa na paliwo chemiczne :)

Reply:
```suggestion
  <ChemfuelPoweredGenerator.label>generator na paliwo chemiczne</ChemfuelPoweredGenerator.label>
```

Reply:
W niektórych tekstach może być za długa linijka.

Reply:
Zostańmy na razie przy tym, kiedyś można popróbować.